dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -529,7 +603,16 @@ private String buildSql(
       PageParams pageParams,
       MapSqlParameterSource mapSqlParameterSource,
       User user) {
-    StringBuilder sqlBuilder = new StringBuilder().append("select * from (");
+    StringBuilder sqlBuilder = new StringBuilder("select ");
+    if (TrackerIdScheme.UID
```

Comment:
Myabe I would invert this `if` to be 
`TrackerIdScheme.UID != queryParams.getIdSchemeParams().getDataElementIdScheme().getIdScheme()`
to match the next `if` logic